packages/apollo-cli/src/commands/queries/check.ts

```diff
@@ -1,46 +1,45 @@
-import "apollo-codegen-core/lib/polyfills";
-import { Command, flags } from "@oclif/command";
-import { table, styledJSON } from "heroku-cli-util";
-import * as Listr from "listr";
-import { toPromise, execute } from "apollo-link";
-import { print, GraphQLError } from "graphql";
+import 'apollo-codegen-core/lib/polyfills';
+import { Command, flags } from '@oclif/command';
+import { table, styledJSON } from 'heroku-cli-util';
+import * as Listr from 'listr';
+import { toPromise, execute } from 'apollo-link';
+import { print, GraphQLError } from 'graphql';
 
-import { loadQueryDocuments } from "apollo-codegen-core/lib/loading";
+import { loadQueryDocuments } from 'apollo-codegen-core/lib/loading';
 
-import { engineFlags } from "../../engine-cli";
-import { engineLink, getIdFromKey } from "../../engine";
-import { gitInfo } from "../../git";
-import { VALIDATE_OPERATIONS } from "../../operations/validateOperations";
-import { ChangeType } from "../../printer/ast";
-import { format } from "../schema/check";
-import { resolveDocumentSets } from "../../config";
-import { loadConfigStep } from "../../load-config";
+import { engineFlags } from '../../engine-cli';
```

Comment:
Would it be possible to revert the Prettier changes present in 721429634a8a41768a86363f3c781fc8485eefca?  We should run it on the whole repository outside the scope of this PR if that's what we want, but for the sake of accurately reviewing this PR it should be kept as a separate concern.